src/features/user/Settings/EditProfile/EditProfileForm.tsx

```diff
@@ -198,7 +219,7 @@ export default function EditProfile({}: Props) {
     onDropAccepted: () => {},
   });
 
-  const saveProfile = async (data: any) => {
+  const saveProfile = async (data: object) => {
```

Comment:
Avoid using `object` or similar generic types. In this case, you could define the type of data submitted by the form.